src/pages/projectSettings/tabs/NotificationsTab/transformers.ts

```diff
@@ -166,10 +176,11 @@ export const formToGql: FormToGqlFunction<Tab> = (
   formState: FormState,
   projectId
 ) => {
-  const { buildBreakSettings, subscriptions } = formState;
+  const { buildBreakSettings, subscriptions, banner } = formState;
   const projectRef: ProjectInput = {
     id: projectId,
     notifyOnBuildFailure: buildBreakSettings.notifyOnBuildFailure,
+    ...(banner && { banner: { ...banner.bannerData } }),
```

Comment:
[nit] I think this spread is redundant
```suggestion
    ...(banner && { banner: banner.bannerData }),
```